src/hiveboard_bridge/src/main.cpp

```diff
@@ -1,19 +1,24 @@
 #include "hiveboard_bridge/MessageHandler.h"
+#include "hiveboard_bridge/ReceiveAction.h"
 #include "hiveboard_bridge/TCPServer.h"
-#include "hiveboard_bridge/ThreadWrapper.h"
 #include "ros/ros.h"
+#include "swarmus_ros_navigation/MoveByMessage.h"
+#include <chrono>
 #include <functional>
 #include <hivemind-host/FunctionCallArgumentDTO.h>
 #include <hivemind-host/FunctionCallRequestDTO.h>
 #include <hivemind-host/FunctionCallResponseDTO.h>
 #include <hivemind-host/HiveMindHostDeserializer.h>
+#include <hivemind-host/HiveMindHostSerializer.h>
 #include <hivemind-host/MessageDTO.h>
 #include <hivemind-host/RequestDTO.h>
-#include <mutex>
+#include <hivemind-host/ResponseDTO.h>
 #include <optional>
 #include <thread>
 
 #define DEFAULT_TCP_SERVER_PORT 8080
+#define DEFAULT_ROBOT_NAME "pioneer_0"
```

Comment:
I'm curious, what would happen if there are multiple pioneer_0? They would both get the messages?

Reply:
You could a constexpr string_view instead (string view are faster to manipulate). I personnaly try to avoid defines, and use constexpr since they are typed, thus easier to debug

Reply:
That's a good question. I guess that both robots would get the messages addressed to "pioneer_0".